catalog/bucket/setters.yaml

```diff
@@ -19,5 +19,5 @@ metadata:
 data:
   name: bucket2
   namespace: config-control
-  project-id: $DEFAULT_PROJECT
+  project-id:
```

Comment:
We should keep `project-id` as the value rather than empty to match other blueprints

Reply:
Updated the value to be null.

Reply:
Done.